### PR TITLE
Dockerfiles reorganization

### PR DESCRIPTION
**Pull request by dustymabe**

### CODE REVIEW

**Comment by cdrage** [comment]
Freakin' awesome.

Also @LalatenduMohanty https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2015-c73b6bdc36 just reached stable :dancer: yay.

Getting error's with Dockerfile.centos in Dockerfiles.pkgs though.
```
dev/atomicapp/Dockerfiles.pkgs  dusty-dockerfiles ✔                                                                                                                                                                                                   14h5m  ⍉
▶ docker build -t atomicapp/dev -f Dockerfile.centos .

▶ aad run foo
Traceback (most recent call last):
  File "/usr/bin/atomicapp", line 5, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib/python2.7/site-packages/pkg_resources.py", line 3011, in <module>
    parse_requirements(__requires__), Environment()
  File "/usr/lib/python2.7/site-packages/pkg_resources.py", line 626, in resolve
    raise DistributionNotFound(req)
pkg_resources.DistributionNotFound: lockfile==0.10.2
```

**Comment by cdrage** [comment]
Dockerfile.fedora (23) in Dockerfiles.pkgs builds fine though!
```
dev/atomicapp/Dockerfiles.pkgs  pull-411 ✔                                                                                                                                                                                                            14h10m  
▶ aad run foo
2015-11-20 16:35:49,731 - atomicapp.install - INFO - App name is foo, will be populated to /atomicapp
2015-11-20 16:35:49,731 - atomicapp.utils - INFO - atomicapp.status.info.message=Loading app foo .
2015-11-20 16:35:49,731 - atomicapp.utils - INFO - atomicapp.status.answer.message={"general": {"namespace": "default", "provider": "kubernetes"}}
2015-11-20 16:35:50,145 - atomicapp.utils - INFO - atomicapp.status.info.message=Pulling image foo ...
Using default tag: latest
Pulling repository docker.io/library/foo
Error: image library/foo:latest not found
2015-11-20 16:35:51,439 - atomicapp.utils - INFO - atomicapp.status.error.message=Couldn't pull foo.
2015-11-20 16:35:51,439 - atomicapp.cli.main - ERROR - Exception caught: Exception("Couldn't pull foo",)
2015-11-20 16:35:51,440 - atomicapp.cli.main - ERROR - Run the command again with -v option to get more information.
```

**Comment by dustymabe** [comment]
@cdrage right. I think 0.1.12 in epel (for centos) has a bug. @cdrage can you review all of these changes considering your release script? 

@cdrage also keep in mind that I'd like to keep the versions in the Dockerfiles.pkgs files to match the versions of the rpms in the repos. So since 0.2.2 isn't in epel/fedora yet then we still have version 0.1.12 there.

**Comment by cdrage** [comment]
@dustymabe will do :) I'll update my hacky bash script and add that to a gist.

**Comment by LalatenduMohanty** [comment]
@dustymabe @cdrage do we really need the dockerfile for RHEL in here? I think CentOS7 is good enough for (as in community)

File: Dockerfiles.pkgs/Dockerfile.rhel7 [hunk 1]
```diff
@@ -9,27 +19,12 @@ LABEL io.projectatomic.nulecule.atomicappversion="0.2.2" \
...
-# add all of Atomic App's files to the container image
-ADD atomicapp/ /opt/atomicapp/atomicapp/
-ADD setup.py requirements.txt /opt/atomicapp/
+WORKDIR /atomicapp                                                                                                                                                                                                                                                             
+VOLUME /atomicapp
 
-# let's install atomicapp
-RUN python setup.py install
+RUN yum install -y atomicapp --setopt=tsflags=nodocs && \
+    yum clean all
```
Comment (LalatenduMohanty) [inline Dockerfiles.pkgs/Dockerfile.rhel7]: This not going to work on a typical RHEL7 machine

**Comment by dustymabe** [comment]
@LalatenduMohanty ok will remove the rhel7 dockerfile

**Comment by dustymabe** [comment]
Fixed comments from review. Merging now

**Pull request merged**